packages/react-components/react-breadcrumb/docs/Spec.md

```diff
@@ -2,54 +2,479 @@
+- Non-clickable content
+- Dropdown Menu
+- Icon (optional)
+
+It can contain a tooltip.
+
+#### DOM
+
+Non-clickable element
+
+```HTML
+<li>
+  {children}
+</li>
+```
+
+Link
+
+```HTML
+<li>
+  <a href="#">
+    {children}
+  </a>
+</li>
+```
+
+Button
+
+```HTML
+<li>
+  <button>
+    {children}
+  </button>
+</li>
+```
+
+Usage
+
+```jsx
+<BreadcrumbItem>
+  Item 1
+</BreadcrumbItem>
+<BreadcrumbItem onClick={() => console.log('smth...')}>
+  Item 2
+</BreadcrumbItem>
+<BreadcrumbItem href="#">
+  Item 3
+</BreadcrumbItem>
+```
+
+#### API
+
+| Property     | Values                                  | Default   | Purpose                                     |
+| ------------ | --------------------------------------- | --------- | ------------------------------------------- |
+| active       | boolean                                 | false     | Indicates if the item is active             |
+| disabled     | boolean                                 | false     | Disables Breadcrumb item                    |
+| divider      | _slot_                                  | `chevron` | Sets type of divider                        |
+| focusMode    | `off`, `no-tab`, `tab-exit`, `tab-only` | `off`     | Sets the focus behavior for the Breadcrumb. |
+| icon         | _slot_                                  |           | Sets icon                                   |
+| iconPosition | `before`, `after`                       | `before`  | Sets icon position                          |
+| tooltip      | _slot_                                  | undefined | Sets custom tooltip                         |
+
+#### Breadcrumb icon
+
+An icon can be SVG or image.
+The icon might be added as a prop:
+
+```jsx
+<BreadcrumbItem icon={<IconComponent />}>Item 1</BreadcrumbItem>
+<BreadcrumbItem icon="../path/name.png">Item 1</BreadcrumbItem>
+<BreadcrumbItem>
+  <SomeSVG />
+  Item 1
+</BreadcrumbItem>
+```
+
+### BreadcrumbDivider
+
+#### DOM
+
+```HTML
+<li aria-hidden="true">
+  {children}
+</li>
+```
+
+#### API
+
+| Property | Values                               | Default   | Purpose              |
+| -------- | ------------------------------------ | --------- | -------------------- |
+| divider  | `chevron`, `slash`, custom component | `chevron` | Sets type of divider |
+
+### BreadcrumbMenu
+
+Dropdown menu is used for collapsed items and subfolders.
+BreadcrumbMenu uses @fluentui/react-menu component.
+
+### BreadcrumbLink
+
+For Buttons and Link @fluentui/react-link component is used.
+If `href` prop is provided it's rendered as a Link, otherwise as Button.
 
 ## Migration
 
-_Describe what will need to be done to upgrade from the existing implementations:_
+### Fabric (v8) property mapping
+
+This should be moved to MIGRATION.md later.
+Here's how the API of v8's `Breadcrumb` compares to the one from v9's `Breadcrumb` component:
+
+#### Props that remain as is
+
+- `maxDisplayedItems`
+- `overflowIndex` => Optional index where overflow items will be collapsed.
```

Comment:
Are you going to use `@fluentui/react-overflow` to implement these? If so, please check if it maps to its API.